### DISCUSSION TITLE
Cannot use Quick Sync

### DISCUSSION THREAD

**Discussion by James4049** [jlesage/docker-handbrake#291] [Q&A]
So im trying to convert my Plex library to H265 10 bit with QVC, I put my files into the "watch" folder and the converted files go into "output" but all the files show the file size as 0 kb and cannot be played but the log shows it was Succesful. I also tried H265 QVC and H264 QVC and 10 bit anything with QVC it does the same thing.

Im runnig Handbrake in Portainer on Openmediavault
CPU is a Intel i3 7100T

I also used this guide to enable quick sync and added the following code into devices
[https://forum.openmediavault.org/index.php?thread/38696-how-to-activate-intel-quick-sync-in-docker-jellyfin-handbrake/](url)

`devices:
- /dev/dri/renderD128:/dev/dri/renderD128
- /dev/dri/card0:/dev/dri/card0`

**Comment by jlesage** [comment]
Do you have the same issue when using the GUI ?  Note that there are still few QSV issues with some CPUs.  Adding `lowpower=0` to VideoOptionExtra in the presets.json fixed the issue for some people.
